src/article/TextureJATS.rng

```diff
@@ -756,7 +756,16 @@
     </element>
   </define>
 
-
+  <!--
+    Allow empty <fn-group>
+  -->
+  <define name="fn-group-model">
+    <zeroOrMore>
+      <choice>
+        <ref name="fn-link.class"/>
```

Comment:
There is no point in using a a class `fn-link.class `here, as there is only one element inside it. Better to do <ref name="fn">, that's more readable.